### PR TITLE
profiles: okular: fix "Print to PDF"

### PR DESCRIPTION
**Pull request by gcb**
Fixes #6402.

### CODE REVIEW

File: etc/profile-m-z/okular.profile [hunk 1]
```diff
@@ -58,7 +58,9 @@ protocol unix
 seccomp
 tracelog
 
-private-bin kbuildsycoca4,kdeinit4,lpr,okular,unar,unrar
+private-bin kbuildsycoca4,kdeinit4,lpr,okular,unar,unrar,ps2pdf
```
Comment (glitsj16) [inline etc/profile-m-z/okular.profile:61]: Sort:

```suggestion
private-bin kbuildsycoca4,kdeinit4,lpr,okular,ps2pdf,unar,unrar
```

File: etc/profile-m-z/okular.profile [hunk 2]
```diff
@@ -58,7 +58,9 @@ protocol unix
...
+# ps2pdf - enables print to pdf functionality
```
Comment (glitsj16) [inline etc/profile-m-z/okular.profile:62]: Remove both this and the following line. We have the outdated info in https://github.com/netblue30/firejail/discussions/6402 if we want to change that in the future.

**Review by glitsj16 — CHANGES_REQUESTED** [review]
Needs minor changes.

File: etc/profile-m-z/okular.profile (hunk 2 above)
Comment (gcb) [reply etc/profile-m-z/okular.profile:62]: k. i usually comment all the things I allow on my app armor profiles. was just going along here too.

**Review by glitsj16 — APPROVED** [review]
LGTM

**Comment by gcb** [comment]
it failed on the sort step which you just suggested to fix

**Review by glitsj16 — APPROVED** [review]
LGTM. Let's run the checks again...

**Review by kmk3 — APPROVED** [review]

**Pull request merged**